diff --git a/env/HERA.env b/env/HERA.env
index 36a6133a..e50f9c79 100755
--- a/env/HERA.env
+++ b/env/HERA.env
@@ -9,7 +9,10 @@ fi

 step=$1

-source ~/venv/global-workflow/bin/activate
+#source ~/venv/global-workflow/bin/activate
+export PATH=/scratch3/NCEPDEV/stmp/Benjamin.Koziol/sandbox/miniconda3/envs/benkozi-work/bin:${PATH}
+module use -a /scratch2/NCEPDEV/nwprod/NCEPLIBS/modulefiles
+module load prod_util
 export launcher="srun --mpi=pmi2 -l --export=ALL --hint=nomultithread"
 export mpmd_opt="--multi-prog --output=mpmd.%j.%t.out"
